src/components/Premium/Header.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+
+import { BackIcon, BagIcon, SearchIcon } from '../../assets/icon';
+
+const Header = () => {
+
+  return (
+
+    <St.HeaderContainer>
+      <St.Button type = "button">
+        <img src={BackIcon} alt="되돌아가기 아이콘" />
+      </St.Button>
+      <St.Header> 프리미엄관 </St.Header>
+      <St.Button type = "button">
+        <img src={SearchIcon} alt="검색 아이콘" />
+      </St.Button>
+      <St.Button type = "button">
+        <img src={BagIcon} alt="장바구니 아이콘" />
+      </St.Button>
+    </St.HeaderContainer>
+
+  );
+};
+
+export default Header;
+
+const St = {
+  HeaderContainer: styled.div`
+    display: flex;
+
+    width: 100%;
+    height: 4.6rem;
+    margin: 0rem;
+    
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.05);
+  `,
+
+  Header: styled.header`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 6.7rem;
+    height: 2.2rem;
+    margin: 1.2rem 5.8rem 1.2rem 10.6rem;
+
+    ${({ theme }) => theme.fonts.SubHead2};
+    ${({ theme }) => theme.colors.gray_900};
+    font-size: 1.6rem;
+    letter-spacing: -0.04em;
+  `,
+
+  Button: styled.button`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+    width: 2.4rem;
+    height: 2.4rem;
+    margin: 1.2rem;
```

Comment:
요기는 이렇게 주면 버튼 영역이 이미지보다 작아져버려서!!(개발자 도구에서 영역 보면 무슨 말인지 알구야!)

```suggestion
    width:4.8rem;
    height: 4.8rem;
```
요렇게 해보면 어떨까?!